### PR TITLE
[RLlib] Terminated/Truncated in Quickstart Script

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
Signed-off-by: Artur Niederfahrenhorst <artur@anyscale.com>

## Why are these changes needed?

Our quickstart example in the docs has been updated to fit gymnasium, but a docstring is missing the truncated flag and all terminated flags are still called done.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by ArturNiederfahrenhorst** [comment]
Thought I had to fix stuff related to https://github.com/ray-project/ray/issues/31321 and https://github.com/ray-project/ray/issues/31314 . That's already fixed though so I ended up with these minor changes in comments @sven1977 .

File: rllib/examples/documentation/rllib_on_ray_readme.py [hunk 1]
```diff
@@ -78,15 +82,15 @@ def step(self, action):
 env = SimpleCorridor({"corridor_length": 10})
 # Get the initial observation (should be: [0.0] for the starting position).
 obs, info = env.reset()
-done = False
+terminated = False
 total_reward = 0.0
 # Play one episode.
-while not done:
+while not terminated:
```
Comment (sven1977) [inline rllib/examples/documentation/rllib_on_ray_readme.py:88]:
```suggestion
while not terminated and not truncated:
```

File: rllib/examples/documentation/rllib_on_ray_readme.py [hunk 2]
```diff
@@ -78,15 +82,15 @@ def step(self, action):
 env = SimpleCorridor({"corridor_length": 10})
 # Get the initial observation (should be: [0.0] for the starting position).
 obs, info = env.reset()
-done = False
+terminated = False
```
Comment (sven1977) [inline rllib/examples/documentation/rllib_on_ray_readme.py:85]:
```suggestion
terminated = truncated = False
```

**Review by sven1977 — APPROVED** [review]
LGTM! Thanks for the cleanup.
Probably more in the docs. I have a docs PR for all the gymnasium changes coming up. :)

**Comment by ArturNiederfahrenhorst** [comment]
Thanks!

File: rllib/examples/documentation/rllib_on_ray_readme.py [hunk 3]
```diff
@@ -24,28 +24,32 @@ def __init__(self, config):
...
         # Walk left.
         if action == 0 and self.cur_pos > 0:
             self.cur_pos -= 1
         # Walk right.
         elif action == 1:
             self.cur_pos += 1
-        # Set `done` and `truncated` flags when end of corridor (goal) reached.
-        done = truncated = self.cur_pos >= self.end_pos
+        # Set `terminated` and `truncated` flag when end of corridor (goal) reached.
+        terminated = truncated = self.cur_pos >= self.end_pos
```
Comment (sven1977) [inline rllib/examples/documentation/rllib_on_ray_readme.py:49]:
```suggestion
        # Set `terminated` flag when end of corridor (goal) reached.
        terminated = self.cur_pos >= self.end_pos
        truncated = False
```

**Pull request merged**